File: app/src/main/java/com/parabola/newtone/mvp/view/RenamePlaylistView.java
package com.parabola.newtone.mvp.view;

import moxy.MvpView;
import moxy.viewstate.strategy.alias.OneExecution;

@OneExecution
public interface RenamePlaylistView extends MvpView {

    void focusOnEditText();

    void setPlaylistTitle(String playlistTitle);
    void setTitleSelected();

    void showToast(String toastText);

    void closeScreen();
    void setPlaylistTitleIsEmptyError();
}

File: app/src/main/java/com/parabola/newtone/mvp/view/RenamePlaylistView.kt
package com.parabola.newtone.mvp.view

import moxy.MvpView
import moxy.viewstate.strategy.alias.OneExecution

@OneExecution
interface RenamePlaylistView : MvpView {
    fun focusOnInputField()

    fun setPlaylistTitle(playlistTitle: String)
    fun setTitleSelected()

    fun closeScreen()

    fun showPlaylistTitleIsEmptyError()
    fun showPlaylistTitleAlreadyExistsError()
}
